lib/obs_github_deployments/cli/commands/is_locked.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module ObsGithubDeployments
+  module CLI
+    module Commands
+      class IsLocked < Dry::CLI::Command
+        # desc "Checks if the deployment is locked"
```

Comment:
remove this comment.

Reply:
That's the description that is going to appear in the help. Better uncomment than remove.

Reply:
I agree with @saraycp